src/main/java/Duke.java

```diff
@@ -1,10 +1,175 @@
+import java.io.*;
+import java.util.*;
+
 public class Duke {
+    enum Request {
+        BYE, LIST, MARK, TODO, DEADLINE, EVENT, DELETE, INVALID
+    }
+
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Hao Wen\n" + "What can I do for you?");
+        //System.out.println("Bye. Hope to see you again soon!");
+
+        MyList myList = new MyList();
+        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+        File file = new File("src/main/java/data/duke.txt");
+
+        try {
+            try (BufferedReader reader = new BufferedReader(new FileReader(file))) {
+                String line;
+
+                while ((line = reader.readLine()) != null) {
+                    String[] loadInput = line.split("\\|");
+                    String type = loadInput[0].trim();
+                    String done = loadInput[1].trim();
+                    String s, s1;
+                    Task task;
+
+                    switch (type) {
+                        case "T":
+                            s = loadInput[2].trim();
+                            task = new Todo(s);
+
+                            if (done.equals("1")) {
+                                task.markAsDone();
+                            }
+
+                            myList.addItem(task);
+                            break;
+                        case "D":
+                            s = loadInput[2].trim();
+                            s1 = loadInput[3].trim();
+                            task = new Deadline(s, s1);
+
+                            if (done.equals("1")) {
+                                task.markAsDone();
+                            }
+
+                            myList.addItem(task);
+                            break;
+                        case "E":
+                            s = loadInput[2].trim();
+                            s1 = loadInput[3].trim();
+                            String s2 = loadInput[4].trim();
+                            task = new Event(s, s1, s2);
+
+                            if (done.equals("1")) {
+                                task.markAsDone();
+                            }
+
+                            myList.addItem(task);
+                            break;
+                    }
+                }
```

Comment:
Following our coding standards, `case` clauses **should not** have indentation in switch statements.